RInAppMessaging/Classes/Views/FullView.swift

```diff
@@ -108,6 +109,12 @@ internal class FullView: UIView, FullViewType, RichContentBrowsable {
         updateUIConstants()
         exitButtonYPositionConstraint.constant = uiConstants.exitButtonVerticalOffset
         bodyViewOffsetYConstraint.constant = hasImage ? 0 : uiConstants.bodyViewSafeAreaOffsetY
+
+        DispatchQueue.main.async { [self] in
+            // Fixes a problem with content size width being set 0.5pt too much
+            // (landscape iPad), resulting in horizontal scroll bouncing.
+            contentScrollView.contentSize.width = contentScrollView.bounds.width
```

Comment:
Should we guard this inside an iPad landscape mode check?
I guess the guard isn't needed if this change works on all devices.

Reply:
Yeah, it works on all devices. Actually even if the content size is set to zero, it still displays normally (scrolling is not working though). I'm not sure if that problem was also visible on iPhone X Max in landscape for example